suite/tests/CMakeLists.txt

```diff
@@ -1031,6 +1031,11 @@ function(template2expect outexpect template runops key)
     # so currently this is unused.
     set(rundefs "${rundefs} -DCYGWIN")
   endif ()
+  if (DEFINED ${key}_runavx512)
+    set(rundefs "${rundefs} -D__AVX__ -D__AVX512F__")
+  elseif (DEFINED ${key}_runavx)
+    set(rundefs "${rundefs} -D__AVX__")
```

Comment:
These are only for template files, but I don't see any template file uses here?  Is this for future use?

Reply:
See suite/tests/linux/sigcontext.template as part of this PR. The AVX512F flag is not used yet.

Reply:
My bad, missed that.